spec/models/active_shipping_calculator_spec.rb

```diff
@@ -108,11 +106,11 @@ module ActiveShipping
 
     describe "compute" do
```

Comment:
Would be nice if we can define the `subject` for this block to be `subject { order.compute(package) }`. That way we can just call `subject` or use the `it { is_expected.to ... }` syntax.